tests/golden/rename-command/rename-command.bats

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bats
+
+load '../helpers/bats-support/load'
+load '../helpers/bats-assert/load'
+load '../helpers'
+
+@test "rename an entry" {
+  run coffer create /a
+  assert_output "[SUCCESS] Entry created at '/a'."
+
+  run coffer rename /a /b
+  assert_output "[SUCCESS] Renamed '/a' to '/b'."
+
+  run cleanOutput coffer view /
+  cat rename-an-entry.gold | assert_output -
+}
+
+@test "rename a directory" {
+  run coffer create /a/b/c
+  assert_output "[SUCCESS] Entry created at '/a/b/c'."
+
+  run coffer rename /a/b /d
+  assert_output "[SUCCESS] Renamed '/a/b/c' to '/d/c'."
+
+  run cleanOutput coffer view /
+  cat rename-a-directory.gold | assert_output -
+}
+
+@test "trying to rename without force" {
+  run coffer create /a/b/c
+  assert_output "[SUCCESS] Entry created at '/a/b/c'."
+
+  run coffer create /d/c
+  assert_output "[SUCCESS] Entry created at '/d/c'."
+
+  run coffer rename /a/b /d
+
+  assert_failure
+  cat rename-without-force.gold | assert_output --partial -
+}
+
+@test "trying to rename with force" {
+  run coffer create /a/b/c
+  assert_output "[SUCCESS] Entry created at '/a/b/c'."
+
+  run coffer create /d/c
+  assert_output "[SUCCESS] Entry created at '/d/c'."
+
+  run coffer rename /a/b /d -f
+  assert_output "[SUCCESS] Renamed '/a/b/c' to '/d/c'."
+
+  run cleanOutput coffer view /
+  cat rename-with-force.gold | assert_output -
```

Comment:
Checking that `view` outputs a single `/d/c` entry doesn't necessarily prove that `/a/b/c` has been renamed to `/d/c/`.

If there was a bug in `coffer rename` that simply deleted the `/a/b/c` entry without copying it, the test would still pass.

We should make this test more robust. I suggest we add a field named `x` to `/a/b/c`, and a field named `y` to `/d/c`. Then, at the end, we check that `view` returns an entry at `/d/c` _with a field named `x`_.

Reply:
Done